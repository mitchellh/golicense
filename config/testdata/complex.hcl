allow = ["one",
  "two", "three/four"
]

deny = [
    "five",
]

override = {
   "six" = "seven"
}

translate = {
    "eight" = "nine"
    "/gopkg.in/(.*)/" = "github.com/\\1"
}
