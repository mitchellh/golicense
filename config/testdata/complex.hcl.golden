(*config.Config)({
 Allow: ([]string) (len=3 cap=3) {
  (string) (len=3) "one",
  (string) (len=3) "two",
  (string) (len=10) "three/four"
 },
 Deny: ([]string) (len=1 cap=1) {
  (string) (len=4) "five"
 },
 Override: (map[string]string) (len=1) {
  (string) (len=3) "six": (string) (len=5) "seven"
 },
 Translate: (map[string]string) (len=2) {
  (string) (len=15) "/gopkg.in/(.*)/": (string) (len=13) "github.com/\\1",
  (string) (len=5) "eight": (string) (len=4) "nine"
 }
})
